### ISSUE TITLE
supported layer mismatch between layer_factory.cpp and feather_convert_caffe.cc

### ISSUE THREAD

**Issue by helloearth012** [Tencent/FeatherCNN#41]
These are what are supported in layer_factory.cpp
void register_layer_creators()
{
REGISTER_LAYER_CREATOR(Input, GetInputLayer);
REGISTER_LAYER_CREATOR(Convolution, GetConvolutionLayer);
REGISTER_LAYER_CREATOR(DepthwiseConvolution, GetDepthwiseConvolutionLayer);
REGISTER_LAYER_CREATOR(BatchNorm, GetBatchNormLayer);
REGISTER_LAYER_CREATOR(LRN, GetLRNLayer);
REGISTER_LAYER_CREATOR(Concat, GetConcatLayer);
REGISTER_LAYER_CREATOR(Dropout, GetDropoutLayer);
REGISTER_LAYER_CREATOR(ReLU, GetReluLayer);
REGISTER_LAYER_CREATOR(PReLU, GetPReluLayer);
REGISTER_LAYER_CREATOR(Scale, GetScaleLayer);
REGISTER_LAYER_CREATOR(Slice, GetSliceLayer);
REGISTER_LAYER_CREATOR(Pooling, GetPoolingLayer);
REGISTER_LAYER_CREATOR(Eltwise, GetEltwiseLayer);
REGISTER_LAYER_CREATOR(InnerProduct, GetInnerProductLayer);
REGISTER_LAYER_CREATOR(Softmax, GetSoftmaxLayer);
REGISTER_LAYER_CREATOR(Filter, GetFilterLayer);
REGISTER_LAYER_CREATOR(Reshape, GetReshapeLayer);
}
These are what are supported in 
 feather_convert_caffe.cc

layer_type.compare("Input")
if (layer_type.compare("Convolution") == 0 || (layer_type.compare("DepthwiseConvolution") == 0))
else if (layer_type.compare("LRN") == 0)
else if (layer_type.compare("Pooling") == 0)
else if (layer_type.compare("Interp") == 0)
else if (layer_type.compare("InnerProduct") == 0)
else if (layer_type.compare("Softmax") == 0)
else if (layer_type.compare("Scale") == 0)
else if (layer_type.compare("Eltwise") == 0)
else if (layer_type.compare("Flatten") == 0)
else if (layer_type.compare("Filter") == 0)

There are a few not supported in this parser. Can you confirm only layers Input, Convolution, LRN, Polling, Interp, InnerProduct, Softmax, Scale, Eltwise, Flattern, Filter are supported?
